xml/Microsoft.Aspnet.Snapin/ISnapinAbout.xml

```diff
@@ -42,9 +50,17 @@
         <Parameter Name="pName" Type="System.IntPtr" RefType="out" />
       </Parameters>
       <Docs>
-        <param name="pName">To be added.</param>
-        <summary>To be added.</summary>
-        <remarks>To be added.</remarks>
+        <param name="pName">A return value that contains the provider name of the snap-in and is passed uninitialized.</param>
```

Comment:
nit: snap-in and is passed uninitialized. --> snap-in. This parameter is passed uninitialized.